### ISSUE TITLE
Wrong documentation Groupnormalizer

### ISSUE THREAD

**Issue by terbed** [jdb78/pytorch-forecasting#1247]
https://pytorch-forecasting.readthedocs.io/en/stable/_modules/pytorch_forecasting/data/encoders.html#GroupNormalizer

>         """
>         Group normalizer to normalize a given entry by groups. Can be used as target normalizer.
> 
>         Args:
>             method (str, optional): method to rescale series. Either "standard" (standard scaling) or "robust"
>                 (scale using quantiles 0.25-0.75). Defaults to "standard".
>             method_kwargs (Dict[str, Any], optional): Dictionary of method specific arguments as listed below
>                 * "robust" method: "upper", "lower", "center" quantiles defaulting to 0.75, 0.25 and 0.5
>             groups (List[str], optional): Group names to normalize by. Defaults to [].
>             center (bool, optional): If to center the output to zero. Defaults to True.
>             scale_by_group (bool, optional): If to scale the output by group, i.e. norm is calculated as
>                 ``(group1_norm * group2_norm * ...) ^ (1 / n_groups)``. Defaults to False.
>             transformation (Union[str, Tuple[Callable, Callable]] optional): Transform values before
>                 applying normalizer. Available options are
> 
>                 * None (default): No transformation of values
>                 * log: Estimate in log-space leading to a multiplicative model
**>                 * logp1: Estimate in log-space but add 1 to values before transforming for stability
>                     (e.g. if many small values <<1 are present).**
>                     Note, that inverse transform is still only `torch.exp()` and not `torch.expm1()`.
>                 * logit: Apply logit transformation on values that are between 0 and 1
>                 * count: Apply softplus to output (inverse transformation) and x + 1 to input
>                     (transformation)
>                 * softplus: Apply softplus to output (inverse transformation) and inverse softplus to input
>                     (transformation)
>                 * relu: Apply max(0, x) to output
>                 * Dict[str, Callable] of PyTorch functions that transforms and inversely transforms values.
>                   ``forward`` and ``reverse`` entries are required. ``inverse`` transformation is optional and
>                   should be defined if ``reverse`` is not the inverse of the forward transformation. ``inverse_torch``
>                   can be defined to provide a torch distribution transform for inverse transformations.
> 
>         """

not logp1, it is log1p!
